src/main/java/com/textmessenger/constant/Constants.java

```diff
@@ -0,0 +1,8 @@
+package com.textmessenger.constant;
+
+public class Constants {
+  private Constants() {
+  }
+
+  public static final String WS_PATH = "/queue/messages";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor your code to get this URI from a customizable parameter. ([squid:S1075](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=176&issues=AWXIMKO_ashydt7ks57i&open=AWXIMKO_ashydt7ks57i)

[ISSUE_KEY]: AWXIMKO_ashydt7ks57i